examples/game_of_life/_helpers.py

```diff
@@ -26,9 +26,12 @@ def get_active_pin_map_id(self) -> str:
         Returns:
             The resource id of the pin map that is registered to the pin map service.
         """
-        return self._sequence_context.Execution.RunTimeVariables.GetValString(
-            "NI.MeasurementLink.PinMapId", 0x0
-        )
+        execution = self._sequence_context.Execution.RunTimeVariables
+        if not execution.Exists("NI.MeasurementLink.PinMapId", 0x0):
+            raise RuntimeError(
+                "Failed to retrieve the active pin map ID .Please check the Update Pin Map step ."
```

Comment:
Please correct the space between the word and punctuation

Reply:
You might want to take a look at this [comment](https://github.com/ni/measurementlink-python/pull/719#discussion_r1595235565) :)

Reply:
Noted.